ava-test/cd.js

```diff
@@ -0,0 +1,103 @@
+import test from 'ava';
+import shell from '..';
+import path from 'path';
+import common from '../src/common';
+import fs from 'fs';
+import utils from './utils/utils';
```

Comment:
Can we group these imports together? I like to group node stdlib imports, then dependency imports, then local imports. Something like:
```javascript
import fs from 'fs';
import path from 'path';
import test from 'ava';
import shell from '..';
import common from '../src/common';
import utils from './utils/utils';
```